madlibs/madlibs.py

```diff
@@ -0,0 +1,69 @@
+'''
+This program is an example solution of what your mad libs game could look like! The code is simple implementation of the game. Are there any ways you can improve it? :)
+'''
+
+# Colour class for fancy text :)
+# Syntax: bcolours.BLUE or GREEN + STRING + bcolours.END
+class bcolours:
+    BLUE = '\033[94m'
+    GREEN = '\033[92m'
+    END = '\033[0m'
+
+print("\nWelcome to PETCS Mad Libs!")
+
+keepPlaying = True
```

Comment:
It would make more sense to put these lines on line 49/after all the functions are declared